notebooks/official/experiments/get_started_with_custom_training_autologging_local_script.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>Point out the parameter enable_autolog</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>done</p>